src/renderer/modules/components/Slider.tsx

```diff
@@ -0,0 +1,25 @@
+import { filters, waitForModule } from "../webpack";
+
+interface SliderProps {
+  disabled?: boolean;
+  markers?: number[];
+  stickToMarkers?: boolean;
+  initialValue?: number;
+  defaultValue?: number;
```

Comment:
`defaultValue` doesn't exist

Reply:
`defaultValue` is green marker

![image](https://user-images.githubusercontent.com/62172116/215683658-585eee31-d6ad-4994-9ad8-c90d2bc41d0d.png)